frontend/src/pages/ShootDetails.vue

```diff
@@ -146,20 +146,24 @@ export default {
   methods: {
     ...mapActions([
       'getShootAddonKyma'
-    ])
+    ]),
+    fetchKymaInfo () {
+      if (!this.isKymaFeatureEnabled && this.isKymaAddonEnabled) {
```

Comment:
KymaAddon is disabled but we fetch the KymaInfo?

Reply:
right, will fix it